workbench-core/environments-ui/src/api/environmentTypeConfigs.ts

```diff
@@ -7,7 +7,9 @@ import { httpApiGet } from '@aws/workbench-core-swb-common-ui';
 import useSWR from 'swr';
 import { EnvTypeConfigItem } from '../models/EnvironmentTypeConfig';
 
-const useEnvTypeConfigs = (id: string): {envTypeConfigs: EnvTypeConfigItem[], areEnvTypeConfigsLoading: boolean} => {
+const useEnvTypeConfigs = (
```

Comment:
formatting seem kind of weird here, is this due to the linting?

Reply:
yep, automatic linting change

Reply:
This linting format is a bit unusual, might need to keep an eye on it next time. Why is it reformatting when this hasn't changed

Reply:
I think it was when I merged from develop. It should only lint staged files